### PR TITLE
feat(player_db): add player online status check

### LINKED ISSUE #904: [Request]: Show who online in players tab
### Problem

_No response_

### Solution

example — копия

### Reason

I think it help a bit to know if angry dude what wait you until next launch of the game is online or not.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Primexz**
requires: https://github.com/YimMenu/Translations/pull/12
fixes: https://github.com/YimMenu/YimMenu/issues/904

image

### CODE REVIEW

File: src/services/player_database/player_database_service.cpp [hunk 1]
```diff
@@ -95,4 +97,39 @@ namespace big
...
+		//fetch current stat for each player.. this will need some time.
+		for (auto& item : m_players)
+		{
+			auto& player = item.second;
+			rage::rlGamerHandle player_handle(player.rockstar_id);
+			rage::rlSessionByGamerTaskResult result;
+			bool success = false;
+			rage::rlTaskStatus state{};
+
+			if (g_pointers->m_start_get_session_by_gamer_handle(0, &player_handle, 1, &result, 1, &success, &state))
```
Comment (maybegreat48) [inline src/services/player_database/player_database_service.cpp:120]: You can pass more than one player into that endpoint, 20 max per call. Keep in mind that spamming the SCAPI would probably lead to a ban

You could try creating the request manually, have no clue where to start though

Comment (maybegreat48) [reply src/services/player_database/player_database_service.cpp:120]: And if you manage to intercept the response before it is processed, you would see a hidden field with the player's session type in it

Comment (Primexz) [reply src/services/player_database/player_database_service.cpp:120]: Alright, I will improve the PR soon


EDIT: I was not able to retrieve multiple player details via this method. It is possible that the error is on my side.

**Review by Yimura — APPROVED** [review]
For me it's fine to merge it like this, we'll see what happens eventually.

Have you checked what happens when people have a lot of players in their database?

**Comment by Primexz** [comment]
> For me it's fine to merge it like this, we'll see what happens eventually.
> 
> 
> 
> Have you checked what happens when people have a lot of players in their database?

I have tried the function with about 50 people in the player database 
Of course it takes a while until we have the online status of each player but so far it works.

**Pull request merged**